windows/installer.nsi

```diff
@@ -415,8 +415,8 @@ FunctionEnd
     !system 'cmd.exe /v /c "for /r "${prefix}" %f in (*.*) do \
         @(set "_f=%f" && echo Delete "$INSTDIR\!_f:${prefix}\=!" >> "${files}")"'
 
-    ; Find target folders in reverse order to ensure they can be deleted when empty
-    !system 'cmd.exe /v /c "for /f "usebackq" %d in \
+    ; Find target folders in reverse order to ensure they can be deleted when empty (include delims= to prevents splitting path names when there is a spaces in the path)
```

Comment:
Similar grammar mistake.

Reply:
At least I was consistent :) or copy/paste the same....not sure if that is a good or a bad thing though. Updated now.

Reply:
Thanks!